rubygem/lib/zeus.rb

```diff
@@ -27,19 +27,20 @@ class << self
     # Yup.
     def setup_dummy_tty!
       return if self.dummy_tty
-      master, self.dummy_tty = PTY.send(:open)
+      coordinator, self.dummy_tty = PTY.send(:open)
       Thread.new {
-        loop { master.read(1024) }
+        loop { coordinator.read(1024) }
       }
       STDIN.reopen(dummy_tty)
       STDOUT.reopen(dummy_tty)
     end
 
-    def setup_master_socket!
-      return master_socket if master_socket
+    def setup_coordinator_socket!
+      return coordinator_socket if coordinator_socket
 
-      fd = ENV['ZEUS_MASTER_FD'].to_i
-      self.master_socket = UNIXSocket.for_fd(fd)
```

Comment:
typo on this line (closing paren shouldn't be in the right square bracket)

Reply:
Meh, I had fixed this locally to get tests passing but failed to commit. My bad!